### PR TITLE
added weapon slot rules

### PR DESCRIPTION
**Pull request by bleehu**

### CODE REVIEW

File: Rules Terrain.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                            </p> </td>
+                        </tr>
+                        <tr>
+                            <td>
+                                <img src="img/terrain/Nuclear Sub.png">
+                                <p>Nuclear Submarine</p>
+                            </td>
+                            <td> 0 </td> <td> 0 </td> <td> 0 </td> <td> <p>
+                                The Nuclear Sub has 10 hp. If you destroy the sub, it explodes as if it has a 
+                                generator level = 3 + (2* round#).
```
Comment (1sourcecontrol) [inline Rules Terrain.html:122]: @bleehu Would you be upset if we swapped the raw 3+ from the Nuclear Sub with the 2+ from the generator? Nuclear power plants are typically significantly larger than what exists in our mechs or on a sub/ship.

Comment (bleehu) [reply Rules Terrain.html:122]: It's a game balance difference for the water eating 1 of those dice. If you still prefer me pull it back down, then I will.

Comment (1sourcecontrol) [reply Rules Terrain.html:122]: Ah, I see. That's fine

File: Rules Building.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	<h1>Building and Buying Rules</h1>
+    <div class="subsystemtype">
+        <h2>Buying And Selling</h2>
+        <div class="subsystemPanel">
+            <div class="subsystem">
+                <p class="rules"> <b>Getting Credits:</b>
+                    At the beginning of the Build Phase, each player gets an allowance of new credits 
+                    depending on what turn it is and how well they did in the last round. The <b>Loser</b> 
+                    of the last round gets the full payout. Each player after that gets 5 fewer credits
+                    in the order of people who died.
```
Comment (1sourcecontrol) [inline Rules Building.html]: Here are the card/income rules I wrote for the next mechanized update. I don't remember where the card rules are, so feel free to reword this to fit in this section: "Winner of the round gets 1st pick of 2 hands of cards. Deal 3 + (number of players) cards, pick 1 and pass it down to the next player in the death-order for that round (please do not disclose the cards you saw with anyone until all players have picked from both hands). As a small compensation for getting first pick, the winner gets -10 income, all players in the middle get -5 income, and the first player that lost the round gets the full amount."

File: Rules Building.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            </div>
+        </div>
+    </div>
+    <div class="subsystemtype">
+        <h2>Building Your Mech</h2>
+        <div class="subsystemPanel">
+            <div class="subsystem">
+                <p class="rules">
+                    <b>Building:</b>
+                    It is possible to build a Mech without equiping any weapons because empty weapon groups
```
Comment (1sourcecontrol) [inline Rules Building.html]: >equiping

spell check: equipping

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]

File: Rules Building.html (hunk 2 above)
Comment (bleehu) [reply Rules Building.html]: Cards are discussed in the Game Phases section, I've added notes to both sections.

**Review by 1sourcecontrol — APPROVED** [review]
Ship-it, baby!
